core/capabilities/triggers/logevent/logeventcap/event_trigger-schema.json

```diff
@@ -0,0 +1,75 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/smartcontractkit/chainlink/core/capabilities/triggers/logevent/log-event-trigger",
+    "$defs": {
+        "head": {
+            "type": "object",
+            "properties": {
+                "Height": {
+                    "type": "string",
+                    "minLength": 1
+                },
+                "Hash": {
+                    "type": "string",
+                    "minLength": 42
+                },
+                "Timestamp": {
+                    "type": "integer",
+                    "minimum": 10000
+                }
+            }
+        },
+        "config": {
+            "type": "object",
+            "properties": {
+                "contractName": {
+                    "type": "string",
+                    "minLength": 1
+                },
+                "contractAddress": {
+                    "type": "string",
+                    "minLength": 42
+                },
+                "contractEventName": {
+                    "type": "string",
+                    "minLength": 1
+                },
+                "contractReaderConfig": {
+                    "type": "object",
+                    "properties": {
+                        "contracts": {
+                            "type": "object"
+                        }
+                    },
+                    "required": ["contracts"]
+                }
+            },
+            "required": ["contractName", "contractAddress", "contractEventName", "contractReaderConfig"]
+        },
+        "output": {
+            "type": "object",
+            "properties": {
+                "Cursor": {
+                    "type": "string",
+                    "minLength": 60
```

Comment:
where is this min coming from?  Do we have a standard way to represent these?

Reply:
This is a sample Emitted Log that I received from ChainReader:

```
EmitLog {"version": "unset@unset", "output": {"Cursor":"5-1-0x62d34461b4592f6db32a699c8
f7d6a5fdcc3fc2a0494d45a840b8ca7fdd3102e","Data":{"Arg0":12},"Head":{"Hash":"sl/c/eRxdSHzeN6pw/STd3YuMr3SiBCFBNT6TT0BIH8=","Height":"5","Timestamp":1727714763}}}
```

I took the minLength from that

Reply:
Would Solana and Aptos have that minimum?

Reply:
Changed this to 1